### PR TITLE
Link to MAVLink example on Pixhawk-Arduino setup

### PR DESCRIPTION
**Pull request by jplopezll**
About a  month ago I published an example on using MAVLink to communicate a Pixhawk with an on board Arduino. I think it could be very useful to beginers as it is focused on a practical approach.
I am proposing an addition in the MAVLink examples list and a second one to the Companion Computers list.

### CODE REVIEW

**Comment by auturgy** [comment]
Merging, because I don't want to make edits on @jplopezll 's master branch.  There are a few tidy ups needed, but I'll raise an issue and fix them later.

File: dev/source/docs/companion-computers.rst [hunk 1]
```diff
@@ -5,24 +5,29 @@ Companion Computers
...
 Related topics on this wiki include:
 
 .. toctree::
     :maxdepth: 1
 
     APSync <apsync-intro>
-    Maverick <https://goodrobots.github.io/maverick/#/>
-    FlytOS <flytos>
+    Arduino family <https://discuss.ardupilot.org/t/mavlink-and-arduino-step-by-step/25566/1>
+    BeaglePilot Project <beaglepilot>
```
Comment (auturgy) [inline dev/source/docs/companion-computers.rst]: beaglepilot shouldn't be added

File: dev/source/docs/companion-computers.rst [hunk 2]
```diff
@@ -5,24 +5,29 @@ Companion Computers
...
 
 Companion Computers can be used to interface and communicate with
-ArduPilot on a flight controller using the MAVLink protocol.  By doing
+ArduPilot on a flight controller using the MAVLink protocol. By doing
 this your companion computer gets all the MAVLink data produced by the
 autopilot (including GPS data) and can use it to make intelligent
 decisions during flight. For example, "take a photo when the vehicle is
-at these GPS co-ordinates".
+at these GPS co-ordinates", gather and preprocess information from advanced
+sensors or actuate on lights, auxiliary servos or any other interfaces.
```
Comment (auturgy) [inline dev/source/docs/companion-computers.rst]: this change needs some work.  Companion computers are increasingly used for "Pilot" functions like collision avoidance and navigation

**Comment by Hwurzburg** [comment]
Jplopezll, please rebase and make changes requested and we can review and finalize...thanks

**Comment by Hwurzburg** [comment]
after review during wiki call, the alphabetizing and addition of Arduino link on companion computers was thought worthwhile...need to keep Beagle Bone as a possible companion computer and maintains TOC entry for board...but Mavlink commands already has the information updated...
since rebasing and modifying is difficult due to multiple intervening changes...concepts have been added to new PR #2244 and merged

**Comment by gadavidd** [comment]
Hey, can you communicate the Pixhawk with an Arduino?

**Comment by auturgy** [comment]
Support requests to the forum please. discuss.ardupilot.org

**Comment by brunoolivieri** [comment]
@gadavidd yes, check on the forum.

**Pull request closed without merging**